MREUnityRuntime/MREUnityRuntimeLib/Core/Components/AnimationComponent.cs

```diff
@@ -37,12 +47,58 @@ internal bool Animating
             }
         }
 
+        private void Update()
+        {
+            // Check for changes to an animation's enabled state and notify the server when a change is detected.
+            var animation = GetUnityAnimationComponent();
+            if (animation)
```

Comment:
>if (animation) [](start = 12, length = 14)

Reduce nesting with an early return, i.e. `if (!animation) return`

Reply:
Lets take stylistic changes offline. Happy to discuss and come up with style guidelines.